src/resources/types.rs

```diff
@@ -1,4 +1,5 @@
 use crate::params::Timestamp;
+use crate::resources::card::{Card, CardBrand, CardType};
```

Comment:
It looks like `Card` wasn't used in the end?  But I'll merge and then make that change on master.